src/main/java/mate/academy/model/ShoppingCart.java

```diff
@@ -15,10 +15,8 @@
 public class ShoppingCart {
     @Id
     private Long id;
-    @OneToMany
-    @JoinTable(name = "shopping_carts_tickets",
-            joinColumns = @JoinColumn(name = "shopping_cart_id"),
-            inverseJoinColumns = @JoinColumn(name = "ticket_id"))
+    @OneToMany(cascade = CascadeType.ALL, orphanRemoval = true)
+    @JoinColumn(name = "shopping_cart_id")
```

Comment:
same, see why `inverseJoinColumns = @JoinColumn(name = "ticket_id"` was here by default

Reply:
why did you remove the correct code?)

Reply:
yes i take too much on myself